drydock_provisioner/ingester/__init__.py

```diff
@@ -104,17 +114,12 @@ def ingest_data(self, plugin_name='', design_state=None, design_id=None, context
                     design_data.add_hardware_profile(m)
                 elif type(m) is node.BaremetalNode:
                     design_data.add_baremetal_node(m)
+                elif type(m) is prom.PromenadeConfig:
```

Comment:
I believe isinstance takes into account inheritance, type does not. I'm looking for exact class, not a set including super classes.